drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -90,7 +94,8 @@ static void nrf5_rx_thread(void *arg1, void *arg2, void *arg3)
 		 * The last 2 bytes contain LQI or FCS, depending if
 		 * automatic CRC handling is enabled or not, respectively.
 		 */
-		if (IS_ENABLED(CONFIG_IEEE802154_RAW_MODE)) {
+		if (IS_ENABLED(CONFIG_IEEE802154_RAW_MODE) ||
+		    IS_ENABLED(CONFIG_NET_L2_OPENTHREAD)) {
```

Comment:
Just make the necessary change so that NET_L2_OPENTHREAD selects RAW MODE, so you won't have to do that check here.

Reply:
Is it a good idea to select raw mode? As raw mode selects NET_RAW_MODE

Reply:
good point, forgot about that. Keep your modification then.

Reply:
We should figure a solution that doesn't require modifying all the drivers to enable openthread.